graylog2-web-interface/src/components/permissions/EntityCreateShareFormGroup.tsx

```diff
@@ -115,11 +118,13 @@ const EntityCreateShareFormGroup = ({
 
     setDisableSubmit(true);
 
+    const prepare_request = isEmpty(newSelectedGranteeCapabilities) ? null: dependenciesGRN;
+
     const payload: EntitySharePayload = {
       selected_grantee_capabilities: newSelectedGranteeCapabilities,
     };
 
-    return EntityShareDomain.prepare(entityType, entityTitle, null, payload).then((response) => {
+    return EntityShareDomain.prepare(entityType, entityTitle, null, { ...payload, prepare_request }).then((response) => {
```

Comment:
same here payload is already defined just before, so we can just add the prepare_request there and only keep payload here.